<launch>
  <node pkg="rosserial_python" type="serial_node.py" name="arduino_motores">
    <param name="port" value="/dev/ttyACM0"/>
  </node>
    <node pkg="rosserial_python" type="serial_node.py" name="arduino_elevadores">
    <param name="port" value="/dev/ttyACM1"/>
  </node>
  <node pkg="rosserial_python" type="serial_node.py" name="arduino_rgb">
    <param name="port" value="/dev/ttyUSB0"/>
  </node>
  <node pkg="rosserial_python" type="serial_node.py" name="arduino_sonar">
<param name="port" value="/dev/ttyUSB1"/>
  </node>
  </launch>